include/zephyr/net/wifi.h

```diff
@@ -318,4 +318,15 @@ static const char * const wifi_twt_setup_cmd2str[] = {
 	[WIFI_TWT_SETUP_CMD_REJECT] = "TWT reject",
 };
 
+enum wifi_twt_setup_resp_status {
+	/* TWT Setup response status */
+	WIFI_TWT_RESP_NOT_RECEIVED = 0,
+	WIFI_TWT_RESP_PROCESSING,
+};
+
+static const char * const wifi_twt_setup_resp2str[] = {
+	/* TWT Setup response status */
+	[WIFI_TWT_RESP_NOT_RECEIVED] = "TWT response not received",
```

Comment:
Do we need to have a string for the processing status?

Reply:
No, it is not required,. only for WIFI_TWT_RESP_NOT_RECEIVED the user will be notified with the message. 
For WIFI_TWT_RESP_PROCESSING, there is already message with the required info.